### PR TITLE
Config page links

### PR DESCRIPTION
**Pull request by Wolveric**
Updating the rep and tickets system with links to their config pages in the control panel, when they error out because the system is disabled.

### CODE REVIEW

File: reputation/plugin_bot.go [hunk 1]
```diff
@@ -35,7 +35,9 @@ func (p *Plugin) BotInit() {
 
 var thanksRegex = regexp.MustCompile(`(?i)( |\n|^)(thanks?\pP*|danks|ty|thx|\+rep|\+ ?\<\@[0-9]*\>)( |\n|$)`)
 
-var repDisabledError = "**Rep command is disabled on this server. Enable it from the control panel.**"
+func repDisabledError(guild *dcmd.GuildContextData) string {
```
Comment (jo3-l) [inline reputation/plugin_bot.go:38]: Perhaps rename this to `createRepDisabledError`?

File: tickets/tickets_commands.go [hunk 2]
```diff
@@ -27,6 +27,10 @@ const InTicketPerms = discordgo.PermissionReadMessageHistory | discordgo.Permiss
 
 var _ commands.CommandProvider = (*Plugin)(nil)
 
+func TicketsDisabledError(guild *dcmd.GuildContextData) string {
```
Comment (jo3-l) [inline tickets/tickets_commands.go:30]: This probably shouldn't be exported. Also, consider renaming this to `createTicketsDisabledError`, similar to above.

**Comment by jo3-l** [comment]
You renamed the functions, but not any of the references. I suggest that at the very least you use an editor/IDE with Go support so these issues are caught before committing.

File: reputation/plugin_bot.go [hunk 3]
```diff
@@ -35,7 +35,9 @@ func (p *Plugin) BotInit() {
 
 var thanksRegex = regexp.MustCompile(`(?i)( |\n|^)(thanks?\pP*|danks|ty|thx|\+rep|\+ ?\<\@[0-9]*\>)( |\n|$)`)
 
-var repDisabledError = "**Rep command is disabled on this server. Enable it from the control panel.**"
+func createRepDisabledError(guild *dcmd.GuildContextData) string {
+	return fmt.Sprintf("**The reputation system is disabled for this server.** Enable it at: <https://yagpdb.xyz/manage/%d/reputation>.", guild.GS.ID)
```
Comment (ashishjh-bst) [inline reputation/plugin_bot.go:39]: instead of using https://yagpdb.xyz, use `web.BaseURL()`

File: tickets/tickets_commands.go [hunk 4]
```diff
@@ -27,6 +27,10 @@ const InTicketPerms = discordgo.PermissionReadMessageHistory | discordgo.Permiss
 
 var _ commands.CommandProvider = (*Plugin)(nil)
 
+func createTicketsDisabledError(guild *dcmd.GuildContextData) string {
+	return fmt.Sprintf("**The tickets system is disabled for this server.** Enable it at: <https://yagpdb.xyz/manage/%d/tickets/settings>.", guild.GS.ID)
```
Comment (ashishjh-bst) [inline tickets/tickets_commands.go:31]: instead of using https://yagpdb.xyz/, use `web.BaseURL()`

**Review by ashishjh-bst — CHANGES_REQUESTED** [review]

**Pull request merged**